contracts/child/governance/ChildGovernor.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/compatibility/GovernorCompatibilityBravoUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesQuorumFractionUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorTimelockControlUpgradeable.sol";
+
+import {NetworkParams} from "contracts/child/NetworkParams.sol";
+
+contract ChildGovernor is
+    GovernorUpgradeable,
+    GovernorCompatibilityBravoUpgradeable,
```

Comment:
We can remove `Bravo` compatibility I think. @gretzke

Reply:
Ack. Waiting.

Reply:
+1 tally supports OZ governor so that should be sufficient imo

Reply:
We've decided we'll update the contract later if necessary.